src/styles/index.js

```diff
@@ -1,18 +1,45 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import * as PropTypes from 'prop-types';
 import { createGlobalStyle, ThemeProvider } from 'styled-components';
 
 const GlobalStyle = createGlobalStyle`
   html {
     font-size:1vw;
     color: ${props => props.theme.primaryColor};
     background: ${props => props.theme.backgroundColor};
-    overflow: hidden;
+  }
+  html, body {
     height: 100%;
+    margin:0;
+    overflow: hidden;
   }
   a {
     text-decoration: none;
   }
+  /* only apply in webkit */
+  @media screen and (-webkit-min-device-pixel-ratio:0) { 
```

Comment:
It will be good if the scroll bar changes only apply to Windows users. Since mac has a built-in feature to hide scroll bar when no action performed.
(refer here)[https://stackoverflow.com/questions/8493589/is-there-a-css-media-query-to-detect-windows]

Reply:
marked. Will do later